src/DataCollection.Shared/ViewModels/FeatureViewModel.cs

```diff
@@ -303,6 +309,45 @@ internal void RaiseFeatureCRUDOperationCompleted(CRUDOperation operation)
                 Args = operation
             });
         }
+
+        /// <summary>
+        /// Uses the Arcade Expression defined in the associated layer's expressions with the title "Subtitle expression" to
+        /// set the <see cref="Subtitle"/>.
+        /// </summary>
+        public async Task ComputeSubtitle()
+        {
+            try
+            {
+                // Evaluate all expressions.
+                var listOfEvaluationResults = await PopupManager.EvaluateExpressionsAsync();
+
+                // Find the matching subtitle expression, if defined.
+                var resultFromCustomExpression = listOfEvaluationResults.FirstOrDefault(res => res.PopupExpression.Title == "Subtitle expression");
+                Subtitle = resultFromCustomExpression?.Result?.ToString();
+            }
+            catch
+            {
+                // Ignore
+            }
+        }
+
+        private string _subtitle = "...";
```

Comment:
Is this the default arcade expression? Should we make the default subtitle string empty? What's the fallback plan here?

```suggestion
        private string _subtitle = "";
```